src/main/java/frc/robot/subsystems/Turret.java

```diff
@@ -0,0 +1,67 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.CONSTANTS;
+
+public class Turret extends SubsystemBase{
+
+    final int kUnitsPerRevolution = 2048; /* this is constant for Talon FX */
+
+    private WPI_TalonSRX turret = new WPI_TalonSRX(2); // config id later
+
+    double kP_turret = 0.1;
+    double kI_turret = 0.0;
+    double kD_turret = 0.0;
+    double kF_turret = 0.0;
+    double kIzone_turrent = 100;
+
+    public Turret(){
+
+        turret.configFactoryDefault();
+        //retrieves the encoders   
+        turret.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative);
+
+        //PID
+        turret.config_kF(2, kF_turret);
```

Comment:
This should probably be a constant in the Constants file, and should be 0 so that it's the default slot.